lib/fluent/rubyprof.rb

```diff
@@ -3,6 +3,16 @@
 
 module Fluent
   class Rubyprof
+
+    PRINTERS = {
+      'flat' => 'FlatPrinter',
+      'flat_with_line_numbers' => 'FlatPrinterWithLineNumbers',
+      'graph' => 'GraphPrinter',
+      'graph_html' => 'GraphHtmlPrinter',
+      'call_stack' => 'CallStackPrinter',
+      'dot' => 'DotPrinter',
```

Comment:
Also, I am wondering of maintenance of this list.
It looks I have to follow if ruby-prof adds more printers.

-------

Maybe, fine. `ruby-prof` is not very active, and new printers are available very rarely, probably.

Reply:
> Why don't you add them

I retrlieved the list of available printes from [bin/ruby-prof](https://github.com/ruby-prof/ruby-prof/blob/master/bin/ruby-prof#L56).

... and I omitted some of them since I could not confirm that they work properly.
Apparently, [there seems to be some unresolved bug in ruby-prof.](https://github.com/ruby-prof/ruby-prof/issues/236)

I'll add them if you want (they will emit error if invoked, though)

Reply:
1. Please add all of printers https://github.com/ruby-prof/ruby-prof/blob/master/bin/ruby-prof#L58-L65
2. I prefer users to specify same option names with https://github.com/ruby-prof/ruby-prof/blob/master/bin/ruby-prof#L58-L65, so it is okay to list up directly (at least for now, I may change mind someday later, though).